tests/src/test/scala/whisk/core/cli/test/WskConfigTests.scala

```diff
@@ -149,9 +149,7 @@ class WskConfigTests
                 // Send request to https://<apihost>/api/v1/namespaces, wsk client passes client certificate to nginx, nginx will
                 // verify it by client ca's openwhisk-client-ca-cert.pem
                 val stdout = wsk.cli(Seq("property", "set", "-i", "--apihost", wskprops.apihost, "--auth", wskprops.authKey,
-                    "--cert", wskprops.cert, "--key", wskprops.key, "--namespace", namespace), env = env).stdout
```

Comment:
@ningyougang FYI above change - when these were added to WskProps, I suspected this would happen since they appear unused with respect to the tests. There is another way to preserve your test without modifying WskProps - perhaps we should do that.